SourceKitStressTester/Sources/StressTester/ActionGenerators.swift

```diff
@@ -211,6 +218,7 @@ public final class TypoActionGenerator: ActionGenerator {
             .codeComplete(offset: contentStart + spelling.new.utf8.count, expectedResult: actionToken.frontExpectedResult),
             .typeContextInfo(offset: contentStart + spelling.new.utf8.count),
             .conformingMethodList(offset: contentStart + spelling.new.utf8.count),
+            .testModule,
```

Comment:
Should this also do the !inFunctionBody thing? Feel free to leave it though. I don't think this generator is enabled by default.

Reply:
Yeah, it should. I'll fix up, thanks.